dali/pipeline/pipeline.h

```diff
@@ -134,13 +134,13 @@ class DLL_PUBLIC Pipeline {
   template<typename T, typename OperatorBackend>
   void
   SetDataSourceHelper(const string &name, const T &tl, OperatorBase *op_ptr, AccessOrder order = {},
-                      InputOperatorSettingMode ext_src_setting_mode = {}) {
+                      InputOperatorSettingMode in_op_setting_mode = {}) {
```

Comment:
What's a "setting mode"? This is not a good name - verbose, but still ambiguous.

Reply:
This has previously been called `ExtSrcSettingMode`.
https://github.com/NVIDIA/DALI/blob/1fee392318c202b82c7ccf4d0bfd60736fd2fd4f/dali/pipeline/operator/builtin/external_source.h#L68

So I just followed the pattern. While this PR might not be a best place to conduct the renaming, if you feel there's a better name I can refactor it in the next PR.